experiments.json

```diff
@@ -28,5 +28,19 @@
       "threshold": 0.3,
       "description": "Show a recommendation in the third highlights spot"
     }
+  },
+  "metadataService": {
+    "name": "Use Mozilla MetadataService",
+    "description": "Compute metadata using MetadataService",
```

Comment:
Add active:false so we can land this disabled until the necessary server infrastructure is deployed and tested.